src/index.js

```diff
@@ -0,0 +1,161 @@
+const state = {
+    currentTempButton: null,
+    increaseTempControl: null,
+    decreaseTempControl: null,
+    landscapeDiv: null,
+    tempValue: null,
+    tempCount: 0,
+    skySelection: null,
+    skyDiv: null,
+    cityNameInput: null,
+    cityNameHeader: null,
+    resetButton: null,
+};
+
+const loadControls = () => {
+    // Retrieve references to all HTML elements we need to access
+    state.increaseTempControl = document.getElementById("increaseTempControl");
+    state.decreaseTempControl = document.getElementById("decreaseTempControl");
+    state.tempValue = document.getElementById("tempValue");
+    state.tempValue.textContent = state.tempCount
+    state.currentTempButton = document.getElementById("currentTempButton");
+    state.skySelection = document.getElementById("skySelect");
+    state.skyDiv = document.getElementById("sky");
+    state.landscapeDiv = document.getElementById("landscape");
+    state.cityNameInput = document.getElementById('cityNameInput');
+    state.cityNameHeader = document.getElementById("headerCityName");
+    state.resetButton = document.getElementById("cityNameReset");
+};
+
+const registerEvents = (event) => {
+    state.increaseTempControl.addEventListener("click", handleIncreaseTempClick);
+    state.increaseTempControl.addEventListener("click", displayLandscape);
+    state.decreaseTempControl.addEventListener("click", displayLandscape);
+    state.decreaseTempControl.addEventListener("click", handleDecreaseTempClick);
+    state.currentTempButton.addEventListener("click", handleCurrentTempClick);
+    state.skySelection.addEventListener("change", displaySky);
+    state.cityNameInput.addEventListener("input", changeCity);
+    state.resetButton.addEventListener("click", resetCityName);
+};
+
+const handleCurrentTempClick = (event) => {
+    displayLandscape();
+    temperatureColor();
+    findLatitudeAndLongitude(state.cityNameInput.value);
+};
+
+const handleIncreaseTempClick = (event) => {
+    // When clicking on the increase temp button, the temp value should increase by 1
+    state.tempCount += 1;
+    state.tempValue.textContent = state.tempCount;
+    temperatureColor();
+};
+
+const handleDecreaseTempClick = (event) => {
+    // When clicking on the decrease temp button, the temp value should decrease by 1
+    state.tempCount -= 1;
+    state.tempValue.textContent = state.tempCount;
+    temperatureColor();
+};
+
+const displayLandscape = (event) => {
+    if (state.tempCount < 59) {
+        state.landscapeDiv.textContent = "🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲";
+    } else if (state.tempCount <= 69) {
+        state.landscapeDiv.textContent = "🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃";
+    } else if (state.tempCount <= 79) {
+        state.landscapeDiv.textContent = "🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷";
+    } else if (state.tempCount >= 80) {
+        state.landscapeDiv.textContent = "🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂";
+}};
+
+const colorChange = (color) => {
+    document.getElementById("tempValue").style.color = color
```

Comment:
While setting inline styles like this works, it's generally not an approach taken in practice. The preferred means of accomplishing this would be to use the classes provided in the stylesheet and toggle the classes via JavaScript.